### PR TITLE
doc: readme and changelog

### PR DESCRIPTION
**Pull request by scnwwu**
Add Readme and Changlog for authentication redirect

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [v0.1.3] - 2023-?
```
Comment (scottdover) [inline CHANGELOG.md:7]: Should we keep this as `Unreleased` until we're actually deploying changes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+
+- Authentication status now persisted in VS Code (#94)
```
Comment (scottdover) [inline CHANGELOG.md:11]: Can we make the issue/pr numbers links to make it easier for folks to get back to the relevant URL?

File: CHANGELOG.md (hunk 1 above)
Comment (scnwwu) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (scnwwu) [reply CHANGELOG.md]: Done. Thanks

**Pull request closed without merging**